### PR TITLE
[TH2-5212] added endpoint to request file by path

### PR DESCRIPTION
**Pull request by molotgor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,6 +105,20 @@ docker compose build
 
 ## Release notes:
 
+### 0.0.5
+
+* added `/file` request for loading content of single jsonl file
+* changed `replacePathServerToLocal` to return empty string in case of not starting with `./notebooks` or `./results`
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:111]: we can say for user. removed ability to get any file from machine via ... REST APIs

File: README.md [hunk 2]
```diff
@@ -105,6 +105,20 @@ docker compose build
...
+
+#### Frontend changes
+
+* added option to change default view type of result group
+* added display of #display-table field in Table view type
+* added option to view last N results of Notebook
+* added validation of Notebook's parameters
+* fixed clearing of Notebook's parameters on run
+* increased width of parameters' inputs
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:120]:
```suggestion
* update local run with jupyter-notebook:
  * updated th2-rpt-viewer:
    * added option to change default view type of result group
    * added display of #display-table field in Table view type
    * added option to view last N results of Notebook
    * added validation of Notebook's parameters
    * fixed clearing of Notebook's parameters on run
    * increased width of parameters' inputs
```

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

File: server.py [hunk 3]
```diff
@@ -153,95 +118,119 @@ def replacePathServerToLocal(path: str):
         return replaceSlashes(path).replace('./notebooks/', notebooksDir, 1)
     elif path.startswith('./results'):
         return replaceSlashes(path).replace('./results/', resultsDir, 1)
-    else:
-        return replaceSlashes(path)
+    raise Exception()
```
Comment (Nikita-Smirnov-Exactpro) [inline server.py:121]: Please add text described the problem

File: server.py [hunk 4]
```diff
@@ -422,10 +442,10 @@ async def reqStop(req: Request):
 
     setup(app)
     app.router.add_route('GET', "/status", reqStatus)
-    app.router.add_route('GET', "/files/all", reqFiles)
     app.router.add_route('GET', "/files/notebooks", reqNotebooks)
     app.router.add_route('GET', "/files/results", reqJsons)
     app.router.add_route('GET', "/files", reqArguments)
```
Comment (Nikita-Smirnov-Exactpro) [inline server.py:447]: Are you sure about endpoint name?

Comment (molotgor) [reply server.py:447]: Yes, this endpoint isn't used on frontend

Comment (Nikita-Smirnov-Exactpro) [reply server.py:447]: maybe remove it ?

Comment (molotgor) [reply server.py:447]: It already removed

**Pull request merged**